packages/renderer/src/lib/ui/ListItemButtonIcon.svelte

```diff
@@ -36,7 +39,17 @@ $: styleClass = detailed
   <DropdownMenuItem title="{title}" icon="{icon}" hidden="{hidden}" onClick="{handleClick}" />
 {:else}
   <!-- enabled button -->
-  <button title="{title}" on:click="{handleClick}" class="{styleClass}" class:hidden="{hidden}">
+  <button
+    title="{title}"
+    on:click="{handleClick}"
+    class="{styleClass} relative"
+    class:disabled="{inProgress}"
```

Comment:
You also have to add a check when invoking the function after the click or use the `disabled` property. Adding the disable class is just to give a visual feedback. If you click twice the start/stop/delete button you can see a double invocation.

Reply:
This existed before, but I agree it's best to fix now. Added a commit to fix this and the temporary flashing back to the action that just completed.